controllers/article_id.controller.js

```diff
@@ -0,0 +1,13 @@
+const fetchArticle = require("../models/article_id.model");
+const getArticle = (req, res, next) => {
+  const id = req.params.article_id;
+  fetchArticle(id)
+    .then(([article]) => {
```

Comment:
We don't need the square brackets here. If all of your tests are passing anyway, you can just take this out before submitting your next PR.